artifacts/snip-dynamic-plugins-support.adoc

```diff
@@ -4,7 +4,9 @@
 
 [IMPORTANT]
 ====
-Red Hat includes a select number of Technology Preview (TP) plugins, available for customers to configure and enable. These plugins and are provided with support scoped per Technical Preview terms, might not be functionally complete, and Red Hat does not recommend using them for production. These features provide early access to upcoming product features, enabling customers to test functionality and provide feedback during the development process.
+{product} includes a select number of Technology Preview plugins, available for customers to configure and enable. These plugins and are provided with support scoped per Technical Preview terms, might not be functionally complete, and {company-name} does not recommend using them for production. These features provide early access to upcoming product features, enabling customers to test functionality and provide feedback during the development process.
```

Comment:
"These plugins and" --> remove "and" so it reads "These plugins are provided"

Does anything about Service Level Agreements (SLAs) need to be mentioned in this disclaimer? For example: "Technology Preview features are not supported with Red Hat production service level agreements (SLAs) and might not be functionally complete."

Reply:
Thanks for your review @gwynnemonahan - I discussed this wording with @reestr and I don't think we need to refer to SLAs.

Reply:
Ah, so legal doesn't require SLAs for Technology Preview for RHDH. Noted.

Reply:
Ah, OK. The links out to https://access.redhat.com/support/offerings/techpreview/ contain info about SLAs. 

Be good to know if SLAs apply to RHDH though. As a user, if I don't click through to https://access.redhat.com/support/offerings/techpreview/ then I won't see the phrase "However, these features are not fully supported under Red Hat Subscription Level Agreements."